scalafix-core/src/main/scala/scalafix/rewrite/NoXml.scala

```diff
@@ -0,0 +1,42 @@
+package scalafix
+package rewrite
+
+import scala.meta._
+import patch.TreePatch.AddGlobalImport
+
+case object NoXml extends Rewrite {
+
+  override def rewrite(ctx: RewriteCtx): Patch = {
+    val patch = ctx.tree.tokens.collect {
+      case tok @ Token.Xml.Start() =>
+        ctx.addLeft(tok, "xml\"\"\"")
+
+      case tok @ Token.Xml.End() =>
+        ctx.addRight(tok, "\"\"\"")
+
+      case tok @ Token.Xml.SpliceStart() =>
+        ctx.addLeft(tok, "$")
+
+      case tok @ Token.Xml.Part(part) =>
+        if (part.contains('$'))
+          ctx.replaceToken(tok, part.replaceAllLiterally("$", "$$"))
+        else Patch.empty
+
+    }.asPatch
+
+    // Emit warning on `{{`
+    ctx.tree.tokens.foreach {
+      case tok @ Token.Xml.Part(part) if part.contains("{{") =>
+        val offset = part.indexOf("{{")
+        val pos =
+          Position.Range(tok.input, tok.start + offset, tok.start + offset + 2)
```

Comment:
I guess I'd like to be able to do something like:
```scala
tok.start.toPos(size = 2)
Position.Range(tok.start, size = 2)
```
I think it is weird to have to pass the input to the Position's constructor when `start: Point` and `end: Point` also carry the input.
WDYT?